doc/changelog/07-commands-and-options/10494-diffs-in-show-proof.rst

```diff
@@ -0,0 +1,3 @@
+- Show proof differences for the "Show Proof" command
```

Comment:
This changelog entry is not clear enough to me. You also forgot to mention that this is assuming that the corresponding flag is set.